docs/operational_guide/placement_changes.md

```diff
@@ -0,0 +1,132 @@
+
+M3Coordinator nodes will also pickup the new placement from etcd and alter which M3DB nodes they issue writse and reads to appropriately.
+
+### Understanding the Placement Configuration
+
+The placement configuration contains a few core values that control how the placement behaves.
+
+#### ID
+
+This is the identifier for a node in the placement and can be any value that uniquely identifies an M3DB node.
+
+#### Isolation Group
+
+This value controls how nodes that own the same M3DB shards are isolated from each other. For example, in a single datacenter configuration this value could be set to the rack that the M3DB node lives on. As a result, the placement will guarantee that nodes that exist on the same rack do not share any shards, allowing the cluster to survive the failure of an entire rack. Alternatively, if M3DB was deployed in an AWS region, the isolation group could be set to the regions availability zone and that would ensure that the cluster would survive the loss of an entire availability zone.
+
+#### Zone
+
+This value controls what etcd zone the M3DB node belongs to.
+
+#### Weight
+
+This value should be an integer and controls how the cluster will weigh the number of shards that an individual node will own. If you're running the M3DB cluster on homogenous hardware, then you probably want to assign all M3DB nodes the same weight so that shards are distributed evenly. On the otherhand, if you're running the cluster on heterogenous hardware, then this value should be higher for nodes with higher resources for whatever the limiting factor is in your cluster setup. For example, if disk space (as opposed to memory or CPU) is the limiting factor in how many shards any given node in your cluster can tolerate, then you could assign a higher value to nodes in your cluster that have larger disks and the placement calcualtions would assign them a higher number of shards.
+
+#### Endpoint
+
+This value should be in the form of <M3DB_HOST_NAME>:<M3DB_NODE_LISTEN_PORT> and identifies how network requests should be routed to this particular node in the placement.
+
+#### Hostname
+
+This value should be in the form of <M3DB_HOST_NAME> and identifies the address / host name of the M3DB node.
+
+#### Port
+
+This value should be in the form of <M3DB_NODE_LISTEN_PORT> and identifies the port over which this M3DB node expects to receive traffic (defaults to 9000).
+
+### Placement Operations
+
+The instructions below all contain sample curl commands, but you can always review the API documentation by navigating to
+
+`http://<M3_COORDINATOR_HOST_NAME>:<CONFIGURED_PORT(default 7201)>/api/v1/openapi`
```

Comment:
Also: https://m3db.io/openapi/

Reply:
nice....how does that get updated

Reply:
Netlify will automatically update this during the [build script](https://github.com/m3db/m3/blob/6b4fe4f57a0db42ee471032b139d62406a25f06c/scripts/site-build.sh#L9) assuming that the assets folder is up to date (via `make asset-gen-query`).